sys/shell/commands/sc_fib.c

```diff
@@ -74,15 +74,33 @@ static void _fib_usage(int info)
 
 static void _fib_add(const char *dest, const char *next, kernel_pid_t pid, uint32_t lifetime)
 {
-    unsigned char *dst = (unsigned char *)dest;
-    size_t dst_size = (strlen(dest));
+    uint32_t prefix = 0;
```

Comment:
The FIB omits the prefix if the specified byte in the falg is 0 **or** the length of the address in bits.
So 0 omits the prefix for any used address type, e.g. IPv6, IPv4, ..., and 128 omits an IPv6 prefix.